storage/blockdevice/COMPONENT_QSPIF/source/QSPIFBlockDevice.cpp

```diff
@@ -1109,6 +1109,13 @@ int QSPIFBlockDevice::_handle_vendor_quirks()
                 // * The SFDP table expects the register bitfield CR3NV[1] to be 1
                 //   but its actual value on the hardware is 0. In order for SFDP parsing
                 //   to work, the quirk reports CR3NV[1] as 1.
+                // * All three possible configurations support 256KB sectors across
```

Comment:
In commit "QSPIFBlockDevice: Add quirk to ignore overlaying sectors on S25FS512S", we say "if we changed them, they would become irreversible." -- could we even change them? If they are OTP, how could we change them? If CR1NV and CR3NV are not programmed, what value would we see?

Reply:
The factory values are 0, and each bit field can be flipped from 0 -> 1 but not 1 -> 0. So the registers are programmable but not erasable. I've updated the commit message.

(I think the idea is, the chip's manufacturer expects the user to do a permanent one-time configuration and stick with it. There's also a register to permanently lock the registers from all further changes (even 0 -> 1), but it's out of the scope and not added to the commit.)